javascript/core.js

```diff
@@ -148,6 +148,7 @@
 
   FourthWall.showLabels = FourthWall.getQueryVariable('labels') === 'true';
 
+  FourthWall.minimumApprovals = 2;
   FourthWall.minimumApprovals = FourthWall.getQueryVariable('approvals');
```

Comment:
So this change actually makes the code worse. The new line at 151 is completely worthless without changing the rest of the code that follows it. I appreciate this has turned into a bit of a slog but I'd like to see a little more thought go into these changes please. Feel free to comment back or Slack me if I'm not being clear enough.

In this case you could just switch the logic of the following `if` to only update `minimumApprovals` if `'approvals'` is *not* undefined. Alternatively you could 'or' the default value with the call to `getQueryVariable` assuming you're confident that it will always return a falsy value or a valid int. Happy to talk this through if you'd find it helpful.